kuzzleio/kuzzle
# Add error message on wrong mapping

## What does this PR do ?

:warning: Depends on https://github.com/kuzzleio/kuzzle/pull/1355

When creating a collection with a mapping or updating a mapping, the user can provide wrong properties (eg: `dinamic`, `foobar`) and Kuzzle will respond `200 OK` since ES just ignore theses properties.

This PR add a check on mapping properties (`dynamic`, `_meta` and `properties` are allowed)

Fix #1352 

### How should this be manually tested?


  - Step 1 : Use this script: 
```
const {
  Kuzzle,
  WebSocket
} = require('kuzzle-sdk');

const kuzzle = new Kuzzle(new WebSocket('localhost'));


(async () => {
  await kuzzle.connect();

  try {
    if (await kuzzle.index.exists('index')) {
      await kuzzle.index.delete('index');
    }
    await kuzzle.index.create('index');
    const rep = await kuzzle.collection.create('index', 'collection', { properties: {
      name: { type: 'keyword' },
      car: {
        meta: 'ee',
        properties: {}
      }
    } })
    console.log(rep)
  } catch (error) {
    console.error(error.message)
  } finally {
    kuzzle.disconnect()
  }
})();
```
  - Step 2 : You should see this error message: `Incorrect mapping mapping property "mapping.properties.car.meta". Did you mean "_meta" ?`

## Other changes

  - dont wrap string at 80 column
